From 062c598c83abd1505d3ea5e980d83033c70b6963 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 27 Jul 2022 07:29:53 +0300
Subject: [PATCH] [SW-95911] gaudi2: sram bist: remove defs

https://gerrit.habana-labs.com/236636 gerrit
adds some macro and enum definitions in common
emb-specs header. So removed those definitions
in preboot header to avoid redundancy.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: Ic66a28d44816e7d31b192796d705251d7c4947a5
---

diff --git a/arch/gaudi2/common/common.h b/arch/gaudi2/common/common.h
index 834d3c7..732d8a0 100644
--- a/arch/gaudi2/common/common.h
+++ b/arch/gaudi2/common/common.h
@@ -200,86 +200,6 @@
 	TPC_ID_SIZE,
 };
 
-enum gaudi2_sram_self_repair_op {
-	SRAM_SELF_REPAIR_OP_BIRA,
-	SRAM_SELF_REPAIR_OP_BISR,
-	SRAM_SELF_REPAIR_OP_BIST
-};
-
-enum gaudi2_sram_ring_type {
-	SRAM_RING_TYPE_TPC,
-	SRAM_RING_TYPE_SOB_ROT,
-	SRAM_RING_TYPE_SRAM,
-	SRAM_RING_TYPE_XBAR_MID,
-	SRAM_RING_TYPE_VDEC,
-	SRAM_RING_TYPE_SOB_XDMA,
-	SRAM_RING_TYPE_NIC,
-	SRAM_RING_TYPE_MME,
-	SRAM_RING_TYPE_HMMU_XBAR,
-	SRAM_RING_TYPE_HMMU_2,
-	SRAM_RING_TYPE_EDMA1_HMMU3,
-	SRAM_RING_TYPE_HMMU_MC,
-	SRAM_RING_TYPE_MAX
-};
-
-#define GAUDI2_RING_PATTERN_VAL0		0
-#define GAUDI2_RING_PATTERN_VAL1		0x1000012
-#define GAUDI2_RING_PATTERN_VAL2		0x121
-#define GAUDI2_RING_PATTERN_VAL3		0x131
-#define GAUDI2_RING_PATTERN_VAL4		0x111
-#define GAUDI2_RING_PATTERN_VAL5		0x1000013
-#define GAUDI2_RING_PATTERN_VAL6		0x9
-#define GAUDI2_RING_PATTERN_VAL7		0x7
-#define GAUDI2_RING_PATTERN_VAL8		0x1000022
-#define GAUDI2_RING_PATTERN_VAL9		0x1
-#define GAUDI2_RING_PATTERN_VAL10		0x100000d
-#define GAUDI2_RING_PATTERN_VAL11		0x1000025
-#define GAUDI2_RING_PATTERN_VAL12		0x1000008
-#define GAUDI2_RING_PATTERN_VAL13		0x85
-#define GAUDI2_RING_PATTERN_VAL14		0x1000006
-#define GAUDI2_RING_PATTERN_VAL15		0xf1
-#define GAUDI2_RING_PATTERN_VAL16		0x109
-#define GAUDI2_RING_PATTERN_VAL17		0x1e1
-#define GAUDI2_RING_PATTERN_VAL18		0x23
-#define GAUDI2_RING_PATTERN_VAL19		0x79
-#define GAUDI2_RING_PATTERN_VAL20		0x100000a
-#define GAUDI2_RING_PATTERN_VAL21		0x3
-#define GAUDI2_RING_PATTERN_VAL22		0x1000001
-#define GAUDI2_RING_PATTERN_VAL23		0x1000072
-#define GAUDI2_RING_PATTERN_VAL24		0x484848
-#define GAUDI2_RING_PATTERN_VAL25		0x48484848
-#define GAUDI2_RING_PATTERN_VAL26		0x12121
-#define GAUDI2_RING_PATTERN_VAL27		0x4c4c4c
-#define GAUDI2_RING_PATTERN_VAL28		0x4c4c4c4c
-#define GAUDI2_RING_PATTERN_VAL29		0x13131
-#define GAUDI2_RING_PATTERN_VAL30		0x444444
-#define GAUDI2_RING_PATTERN_VAL31		0x44444444
-#define GAUDI2_RING_PATTERN_VAL32		0x11111
-#define GAUDI2_RING_PATTERN_VAL33		0x100007f
-#define GAUDI2_RING_PATTERN_VAL34		0x1000031
-#define GAUDI2_RING_PATTERN_VAL35		0x100002a
-#define GAUDI2_RING_PATTERN_VAL36		0x100002e
-#define GAUDI2_RING_PATTERN_VAL37		0x1000016
-#define GAUDI2_RING_PATTERN_VAL38		0x1000032
-#define GAUDI2_RING_PATTERN_VAL39		0x1000037
-#define GAUDI2_RING_PATTERN_VAL40		0x1000019
-#define GAUDI2_RING_PATTERN_VAL41		0x100004a
-#define GAUDI2_RING_PATTERN_VAL42		0x1000052
-#define GAUDI2_RING_PATTERN_VAL43		0x100005b
-#define GAUDI2_RING_PATTERN_VAL44		0x100005a
-#define GAUDI2_RING_PATTERN_VAL45		0x1212121
-#define GAUDI2_RING_PATTERN_VAL46		0x1313131
-#define GAUDI2_RING_PATTERN_VAL47		0x1111111
-#define GAUDI2_RING_PATTERN_VAL48		0x1000064
-#define GAUDI2_RING_PATTERN_VAL49		0x1000028
-#define GAUDI2_RING_PATTERN_VAL50		0x100001a
-#define GAUDI2_RING_PATTERN_VAL51		0x100001c
-#define GAUDI2_RING_PATTERN_VAL52		0x1000010
-#define GAUDI2_RING_PATTERN_VAL53		0x1000042
-#define GAUDI2_RING_PATTERN_VAL54		0x100001f
-#define GAUDI2_RING_PATTERN_VAL55		0x1000049
-
-
 void gaudi2_axi_drain_ena(void);
 uint32_t gaudi2_gpio_read(uint32_t gpio_pin);
 bool preboot_fake_security_is_enabled(void);
